js/src/admin.js

```diff
@@ -2,6 +2,67 @@
  * @package Polylang
  */
 
+window.pll = window.pll || {};
+
+/**
+ * Calls the passed callable when the DOM is ready.
+ *
+ * @param {callable} fn - Function to call when the DOM is ready.
+ */
+window.pll.dom_ready = ( fn ) => {
+	if ( document.readyState !== 'loading' ) {
+		fn();
+	} else {
+		document.addEventListener( 'DOMContentLoaded', fn );
+	}
+};
+
+/**
+ * Settings: manages AJAX calls attached to `.pll-ajax-button` buttons.
+ */
+window.pll.settings_ajax_button = () => {
```

Comment:
I see you have a modular approach so I suggest to go all in.
- Move the code in a dedicated file then import it here.
- Have a `init()` function so you elements are prepared before the event listener callback is called.

Bonus, camelCase please ❤️

Reply:
Done in [411277a](https://github.com/polylang/polylang/pull/1420/commits/411277a5b02769446f93f41ed8cbed2522dddc41) and [9ce6f69](https://github.com/polylang/polylang/pull/1420/commits/9ce6f69003c2b3d8c8f7a5f4d9a3d8a881be5249).